package.json

```diff
@@ -60,8 +60,8 @@
     "redux": "^3.6.0",
     "redux-thunk": "^2.2.0",
     "rimraf": "^2.5.4",
-    "semantic-release": "^6.3.2",
-    "travis-github-status": "^1.5.4",
+    "semantic-release": "^15.9.14",
```

Comment:
wow, thats alot of breaking changes

Reply:
Unfortunately, automated released are not working anymore :P so might as well upgrade it all